src/components/Button/Button.spec.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { ThumbUp } from '@sumup/icons';
+
+import {
```

Comment:
I do prefer your approach of importing the test utils, but we also export them as globals, right? If so, shouldn't we remove them?

Reply:
The reason why I switched to importing the globals is that TypeScript doesn't support it. In hindsight, it's the better approach anyway since it's clearer where the utils are coming from. 

I kept the globals so that we don't have to update all JS spec files. We can remove them once the migration to TypeScript is complete.